### PR TITLE
DS-1041: Deprecate Grid Span

### PR DESCRIPTION
**Pull request by crishpeen**

### CODE REVIEW

File: packages/web-react/src/components/Grid/README.md [hunk 1]
```diff
@@ -58,6 +58,39 @@ Advanced example usage:
 </Grid>
 ```
 
+### ⚠️ DEPRECATION NOTICE
+
+GridSpan component is deprecated and will be removed in the next major version. Use `GridItem` instead.
+
+[What are deprecations?][readme-deprecations]
+
+#### Migration Guide
```
Comment (literat) [inline packages/web-react/src/components/Grid/README.md:67]: Should this be repeated in this readme as well? I mean the same text is in deprecations so maybe just linking it?

File: packages/web-twig/src/Resources/components/Grid/README.md [hunk 2]
```diff
@@ -102,6 +102,35 @@ Without lexer:
 {% endembed %}
 ```
 
+### ⚠️ DEPRECATION NOTICE
+
+GridSpan component is deprecated and will be removed in the next major version. Use `GridItem` instead.
+
+[What are deprecations?][readme-deprecations]
+
+#### Migration Guide
```
Comment (literat) [inline packages/web-twig/src/Resources/components/Grid/README.md:111]: Same here. I suggest to link it to deprecations.

File: packages/web/src/scss/components/Grid/README.md [hunk 3]
```diff
@@ -148,6 +148,29 @@ on larger breakpoints, you can combine Grid Span with responsive Grid columns:
 👉 Keep in mind Grid Span modifiers are designed to work only with a 12-column
 Grid (`.Grid[--cols-12]`, `.Grid--tablet--cols-12`, `.Grid--desktop--cols-12`).
 
+### ⚠️ DEPRECATION NOTICE
+
+GridSpan classes are deprecated and will be removed in the next major version. Use `GridItem` class instead.
+
+#### Migration Guide
```
Comment (literat) [inline packages/web/src/scss/components/Grid/README.md:155]: Here as well.

File: tests/e2e/demo-components-compare.spec.ts-snapshots/grid-chromium-linux.png
Comment (literat) [inline tests/e2e/demo-components-compare.spec.ts-snapshots/grid-chromium-linux.png:1]: Shouldn't it be better to have here the "grid span pyramid" but using the new syntax? From my point of view, this is a loss of information.

File: packages/web-react/DEPRECATIONS-v2.md [hunk 4]
```diff
@@ -194,6 +194,37 @@ Please, add the `danger` icon to your project's assets.
...
 Either install newer version of the `spirit-icons` package or add `danger` named icon to your project's icons.
 
+### GridSpan Component
+
+The `GridSpan` component is deprecated and will be removed in the next major version. Use `GridItem` instead.
+
+#### Migration Guide
+
+Use our codemod to automatically migrate your code.
+
```
[the remark this answers is not in the document]
Comment (adamkudrna) [reply packages/web-react/DEPRECATIONS-v2.md:204]: + link to codemods readme please.

File: packages/web-react/src/components/Grid/README.md (hunk 1 above)
Comment (adamkudrna) [reply packages/web-react/src/components/Grid/README.md:67]: As agreed in person, please keep the migration instructions only in the DEPRECATIONS file. We can leave the deprecation notice here.

**Review by adamkudrna — APPROVED** [review]
Please delete the migration instructions from component READMEs, it's just fine to have it in the package's DEPRECATIONS file. The rest is just minor comments. Good job!

File: .prettierignore [hunk 5]
```diff
@@ -29,3 +29,6 @@ packages/**/*.hbs
 
 # Narwal NX files (cache etc.)
 .nx
+
+# Ignore codemods test outputs, because we compare them with generated code which we cannot format
+packages/codemods/src/transforms/**/*.output.tsx
```
[the remark this answers is not in the document]
Comment (adamkudrna) [reply .prettierignore:34]: My bad, this is prettierignore 🙈.

File: tests/e2e/demo-components-compare.spec.ts-snapshots/grid-chromium-linux.png
Comment (crishpeen) [reply tests/e2e/demo-components-compare.spec.ts-snapshots/grid-chromium-linux.png:1]: I added a simple demo to all packages with a centered responsive grid item.

**Review by literat — APPROVED** [review]
LGTM 👍

**Pull request merged**